jms/1.8/modules/ROOT/pages/jms-troubleshooting.adoc

```diff
@@ -78,6 +78,12 @@ To resolve this scenario:
 
 Implement a fallback solution by managing the life cycle of containers to detect when the primary node fails and starts a new node.
 
+== Troubleshoot Prefetch
+
+Using prefetch can result in the loss of predictability of API calls. If prefetch is used, JMS Connector aggressively tries to fetch messages, which results in increased API calls. If predictability and control on API requests is important, use the following steps to disable prefetch and strictly follow the polling time:
+
```

Comment:
Steps to disable prefetch are different for each JMS provider vendor so there isn't a generic way to do it, we could suggest checking the vendor documentation to learn how to disable it.
There are two ways to consume messages:
1- consume operation (does not support prefetch enabled when ACK MANUAL is configured, otherwise unexpected behavior on the order in which messages are consumed can arise).
2- on new message (for some vendors prefetch enabled is required to use the async APIs)